tests/inputs/TestNode.cpp

```diff
@@ -28,3 +28,21 @@ TEST(TestNode, base) {
   ASSERT_TRUE(*node2 >= *node);
   ASSERT_TRUE(*node0 >= *node);
 }
+
+TEST(TestNode, line) {
+  auto vl = std::make_shared<dfl::inputs::VoltageLevel>("VL");
+  std::vector<dfl::inputs::Shunt> shunts1 = {dfl::inputs::Shunt("1.1")};
```

Comment:
Unit tests are missing for at least transfo and network manager with shunts. Please make sure the overall coverage is as high as possible

Reply:
OK

Reply:
These are covered even without new tests

Reply:
What is the latest code coverage?

Reply:
88% in total, almost 100% for inputs